hudi-common/src/test/java/org/apache/hudi/common/testutils/FileCreateUtils.java

```diff
@@ -59,6 +64,8 @@
 
 public class FileCreateUtils {
```

Comment:
to align with the new design, we should later aim to restrain its use. This can be useful for testing low-level file-manipulation logic. HoodieTestTable should leverage more src code path.